app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt

```diff
@@ -2684,6 +2684,24 @@ class DatabaseMigrationAndroidTest {
     verifyPatientPhoneNumber(dbV42, mobileUuid, "mobile")
     verifyPatientPhoneNumber(dbV42, landlineUuid, "landline")
   }
+
+  @Test(expected = SQLiteException::class)
+  fun drop_communication_table_from_42_to_43() {
```

Comment:
FYI, we have extension functions on the `SUpportSQLiteDatabase` class which let you verify whether a given table exists or not.

So this entire test can become
```kotlin
val dbv42 = helper.createDatabase(42)
dbv42.assertTableExists("Communication")

val dbv43 = helper.createDatabase(43)
dbv43.assertTableDoesNotExist("Communication")
```

Reply:
Nice!